content/applications/finance/fiscal_localizations/hong_kong.rst

```diff
@@ -0,0 +1,70 @@
+=========
+Hong Kong
+=========
+
+Configuration
+=============
+
+:ref:`Install <general/install>` the :guilabel:`Hong Kong - Accounting` module to get the features
+of the HK localization.
+
+.. image:: hong_kong/hk-module.png
+   :align: center
+   :alt: "Hong Kong - Accounting" localization module marked as "installed"
+
+Add FPS QR Codes to invoices
+============================
+
+:abbr:`FPS (Faster Payment System)` is a payment service platform that allows customers to make
+instant domestic payments to individuals and merchants in Hong Kong dollar or Renminbi via online
+and mobile banking.
+
+Activate QR Code feature
+------------------------
+
+Go to :menuselection:`Accounting --> Configuration --> Settings` and then in the :guilabel:`Customer
+Payments` section to activate the :guilabel:`QR Codes` feature.
+
+.. image:: hong_kong/hk-qr-code-setting.png
+   :align: center
+   :alt: "QR Codes" feature activated
+
+FPS bank account configuration
+------------------------------
+
+Go to :menuselection:`Contacts --> Configuration --> Bank Accounts`, select an account Holder with
+Hong Kong as country, open your FPS bank account and fill in the :guilabel:`FPS Type` and
+:guilabel:`FPS ID/Phone Number/Email Address` fields.
+
+.. image:: hong_kong/hk-fps-bank-setting.png
+   :align: center
+   :alt: FPS bank account configuration
+
+.. seealso::
+   :doc:`../accounting/bank/setup/bank_accounts`
+
+Bank Account's journal configuration
+------------------------------------
+
+Go to :menuselection:`Accounting --> Configuration --> Journals`, open your bank journal, then fill
+out the Account Number and Bank under the Journal Entries tab.
+
+.. image:: hong_kong/hk-bank-account-journal-setting.png
+   :align: center
+   :alt: Bank Account's journal configuration
+
+Issue Invoices with FPS QR Codes
+--------------------------------
+
+To enable the FPS QR Code on your invoices, go to :menuselection:`Accounting --> Customers -->
+Invoices`, and click :guilabel:`New` to create a new invoice.
+
+Open the :guilabel:`Other Info` tab and select the  :guilabel:`Payment QR-code` option to *EMV
+Merchant-Presented QR-code*.
+
+.. image:: hong_kong/hk-qr-code-invoice-setting.png
+   :align: center
+   :alt: Select EMV Merchant-Presented QR-code option
+
+Make sure that the :guilabel:`Recipient Bank` is the one you want to use to receive your customer's
+payment as Odoo uses this field to generate the FPS QR Code.
```

Comment:
```suggestion
Ensure that the :guilabel:`Recipient Bank` is the one you configured, as Odoo uses this field to generate the FPS QR code.
```
I would rephrase to clarify that you must select the one you just configured, not something else. If you need to specify "the one you want to use to receive customers' payments", move that info to the "FPS bank account configuration" heading.